gnomad_qc/v4/create_release/validate_and_export_vcf.py

```diff
@@ -12,7 +12,13 @@
     pprint_global_anns,
     validate_release_t,
 )
-from gnomad.resources.grch38.gnomad import POPS, SUBSETS
+from gnomad.resources.grch38.gnomad import (
+    HGDP_POPS,
```

Comment:
I think HGDP_POPS (and for that matter HGDP_POP_NAMES) https://github.com/broadinstitute/gnomad_methods/blob/0d2e71f93d5cfceebfad2ab538e47d8457d57d6d/gnomad/resources/grch38/gnomad.py#L189 need to be updated @KoalaQin ?